core/contracts/composed_stream_template.kf

```diff
@@ -5,14 +5,14 @@
 database composed_stream_db_name;
 
 table taxonomies {
+    taxonomy_id         uuid    primary notnull,
     child_stream_id     text    notnull,
     child_data_provider text    notnull,
     weight              decimal(36,18)   notnull,
     created_at          int     notnull,    //  block height
     disabled_at         int,                //  block height
     version             int     notnull,
-
-    #identifier_idx   primary(version, child_data_provider, child_stream_id)
+    start_date          text    notnull // indicates the start date of when the taxonomy is valid. i.e. when the weight starts to be used
```

Comment:
won't we have that fallback mechanism? I guess not null would prevent this

Reply:
i.e. if we don't have a fallback, we will always need to publish a start_date even for streams that doesn't need to have one.

Reply:
ah right, will make it null then